package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bedrock-vue-address-form",
+  "version": "0.0.1-0",
+  "description": "Vue.js address form component for Bedrock Web apps",
+  "main": "index.js",
+  "less": "main.less",
+  "peerDependencies": {
```

Comment:
Just want to confirm that these are the right kind of dependency.  Does browser `dependencies: all` do anything if there are no `dependencies`?

Reply:
`dependencies: all` includes peer dependencies: https://github.com/digitalbazaar/bedrock-views/blob/master/lib/packages.js#L482-L490

So the above will work. That being said, I'm not sure why they are peer dependencies here instead of regular ones. I'm willing to accept this as is given the circumstances -- we should revisit what our policy is, but I thought we decided to use regular dependencies because the advantages outweighed the disadvantages (with neither being fully ideal options).